database/migrations/2022_03_08_122942_add_eula_to_checkout_acceptance.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddEulaToCheckoutAcceptance extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('checkout_acceptances', function (Blueprint $table) {
+            $table->text('stored_eula')->nullable()->default(null);
+            $table->string('stored_eula_file')->nullable()->default(null);
```

Comment:
We're not actually *doing* anything with this yet, but it makes sense to store the current locked-in-time EULA and the filename (maybe) here in the `checkout_acceptances` table (or maybe the `action_logs` table - I could be convinced.)

We'll need to store those values, and also expose them so that the asset admins and the user who has accepted the item can download them at any time.